### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by cat-markwylde**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+node_modules
+dist/*.min.js
+.DS_Store
```
Comment (votemike) [inline .gitignore:3]: I've always been of the opinion that people should have this in their global gitignore.
Same with IDE config files.

File: demo/index.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+const data = [
+  {
+    name: 'Privacy',
+    heading: <span>Privacy</span>,
+    child: <div>Privacy is <a href="https://apple.com/privacy">very</a> important to Apple.</div>
+  },
+  {
+    name: 'A Picture',
+    heading: 'A Picture',
+    child: <img src="./images/applesOrange.jpeg" />,
```
Comment (votemike) [inline demo/index.tsx:14]: We don't have this image, perhaps that will confuse someone viewing the demo?

File: dist/index.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="shortcut icon" href="data:image/x-icon;," type="image/x-icon"> 
+  <title>React Accordion</title>
+  <script src="index.min.js"></script>
+</head>
+<body>
+  <app></app>
+</body>
+</html>
```
Comment (votemike) [inline dist/index.html:14]: New line?

File: package.json [hunk 4]
```diff
@@ -0,0 +1,44 @@
+{
+  "name": "react-accordion",
+  "version": "1.0.0",
```
Comment (votemike) [inline package.json:3]: If we are going to release this on NPM, we shouldn't make this 1.0.0.
Maybe 0.1.0?

File: package.json [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    "ui"
+  ],
+  "author": "",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/votemike/react-accordion/issues"
+  },
+  "homepage": "https://github.com/votemike/react-accordion#readme",
+  "dependencies": {
+    "react": "^18.2.0"
```
Comment (votemike) [inline package.json:28]: We want to support a few versions of React don't we? Not just the latest version.
Should this be a peer dependency?

File: tsconfig.json [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    "scripts",
+    "acceptance-tests",
+    "webpack",
+    "jest",
+    "src/setupTests.ts"
+  ],
+  "types": [
+    "typePatches"
+  ]
+}
```
Comment (votemike) [inline tsconfig.json:30]: New line?

**Review by votemike — COMMENTED** [review]
What is the difference between demo/, dist/ and react-accordion-test/?
Perhaps something in the README? Or a CONTRIBUTING.MD?

**Comment by markwylde** [comment]
All comments should be resolved now in https://github.com/votemike/react-accordion/pull/3/commits.

Let's get this merged, so we can get the second PR in next.

File: tsconfig.json [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+    "noImplicitThis": true,
+    "noImplicitAny": true,
+    "strictNullChecks": true,
+    "esModuleInterop": true,
+  },
+  "exclude": [
+    "node_modules",
+    "build",
+    "scripts",
+    "acceptance-tests",
```
Comment (votemike) [inline tsconfig.json:22]: Do we have this file/folder?

File: react-accordion-test/package.json [hunk 8]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "react-accordion-test",
```
Comment (votemike) [inline react-accordion-test/package.json:2]: What's this file?

File: tsconfig.json (hunk 7 above)
Comment (markwylde) [reply tsconfig.json]: Ahh nice one. Removed all that.

File: react-accordion-test/package.json (hunk 8 above)
Comment (markwylde) [reply react-accordion-test/package.json:2]: The next PR removes this folder.

We can either merge in the second PR to this one, and then review. Or we can merge this in and then the second one fixes this.

I'm happy either way, but let's make a move on this.

File: react-accordion-test/index.js [hunk 9]
```diff
@@ -0,0 +1 @@
+const a = require('react-accordion');
```
[the remark this answers is not in the document]
Comment (markwylde) [reply react-accordion-test/index.js:1]: The next PR removes this folder.

We can either merge in the second PR to this one, and then review. Or we can merge this in and then the second one fixes this.

I'm happy either way, but let's make a move on this.

File: react-accordion-test/package.json (hunk 8 above)
Comment (markwylde) [reply react-accordion-test/package.json:2]: Merge it! Merge it! Merge it! :tada: :partying_face:

**Review by votemike — APPROVED** [review]

**Pull request merged**